### PR TITLE
Fix Table of Contents overflow

### LINKED ISSUE #4071: Table of contents does not break for Russian language
⚠️ This is for issues with the https://developer.chrome.com site, not Chromium itself.
If you want to file a bug with Chromium (the open-source project behind Google Chrome and other browsers), please use https://crbug.com.

**Describe the bug**
In Russian version of devtools 104 blog post the aside is too big!
I think that because the titles are long. Some CSS fixes needs.

Link: https://developer.chrome.com/ru/blog/new-in-devtools-104/
Screen: 1440px
Browser: Safari 16.1
Hardware: MacBook M1 Pro

**To Reproduce**
1. Go to https://developer.chrome.com/ru/blog/new-in-devtools-104/
1. See error

**Expected behavior**
I see two of three possible fixes:
1. Limit the width of aside and give to titles `nowrap`. Then the end of titles will be under the horizontal scroll. Not the best/
2. Give `text-overflow: ellipsis;` to the titles. Looks good, but can make some difficulties with Ctrl + F search through the page.
3. Limit the width and allow titles to wrap to the next line.

**Screenshots**

CleanShot 2022-10-20 at 13 08 32@2x

**Desktop (please complete the following information):**
 - OS: MacOS Ventura 13.0
 - Browser Safari 
 - Version 16.1

**Additional context**

### PR DESCRIPTION
**Pull request by Truemedia**
Fixes #4071

Changes proposed in this pull request:

Specific to #4071
- Unable to replicate this in problem safari while devtools is closed but when open can see as the scrollbar built-in scrollbar style changes.
- Suggested CSS solution by Dan seems to work even without a max width and text search using CTRL + F still seems to work and the text is only partially visually hidden while retaining browser features around markup (text content).

Other changes specific to TOC functionality for a separate ticket also included (#3097)

### CODE REVIEW

File: site/_includes/partials/toc-under-tree.njk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{#
+  This is used as the TOC on smaller screens (<= 992) that makes use of the white space
+  under the navigation tree when the user scrolls down, in response to:
+  https://github.com/GoogleChrome/developer.chrome.com/issues/3097
+#}
+<div class="toc-container toc-container-under-navigation-tree display-none md:display-block xl:display-none pad-right-400 {% if not tocContents %}visibility-hidden{% endif %}">
```
Comment (devnook) [inline site/_includes/partials/toc-under-tree.njk:6]: Why do you need this template? is it used somewhere?

**Review by devnook — CHANGES_REQUESTED** [review]
I think 11 toc-under-tree.njk is not needed?

File: site/_includes/partials/toc-under-tree.njk (hunk 1 above)
Comment (Truemedia) [reply site/_includes/partials/toc-under-tree.njk:6]: This is not needed and can be removed

**Comment by devnook** [comment]
@Truemedia any chance updating this PR?

**Comment by Truemedia** [comment]
> @Truemedia any chance updating this PR?

That template file has been removed in latest commit on this PR

**Review by devnook — APPROVED** [review]

**Pull request merged**